src/dashboard/post_distribute/post_distribute.py

```diff
@@ -0,0 +1,51 @@
+"""Handler for validating the payload contents and submitting to queue"""
+
+import json
+import os
+
+import boto3
+import requests
+
+from shared import decorators, functions
+
+sns_client = boto3.client("sns", os.environ.get("AWS_REGION"))
+valid_chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-._~:/?#[]@!$&'()*+,"
+
+
+def validate_github_url(config):
+    if not config["url"].startswith("https://github.com/smart-on-fhir/") or any(
+        c not in valid_chars for c in config["url"]
+    ):
```

Comment:
I _think_ this is safe, but I'm not like 100% confident that there isn't some valid URL syntax that lets you have `github.com/smart-on-fhir/` as your username and then the real domain afterward :smile: 

I'd feel lightly safer if you used urllib.parse and then checked netloc and path. But not critical. I couldn't break this with some light testing.

Reply:
if you want - i went looking into urllib.parse's behavior and it specifically does not do the character kind of validation, so I think this is stricter than that? but can't hurt.